helpers/file_helper.rb

```diff
@@ -26,10 +26,10 @@ def self.file_rename(file_name)
   # @return [nil]
   def self.check_temp_dir(dir_name)
     if File.exist? dir_name
-      puts "Directory #{dir_name} exist?: true"
+      OnlyofficeLoggerHelper.green_log "Directory #{dir_name} exist?: true"
```

Comment:
I think this log maybe use default color (use method `log`), since it's not very critical information

Reply:
Yes, I agree. Check for folder existence added